### PR TITLE
Update README URLs based on HTTP redirects

### PR DESCRIPTION
**Pull request by ReadmeCritic**
Created with https://github.com/dkhamsing/frankenstein

### GitHub Corrected URLs 
Was | Now 
--- | --- 
https://github.com/IonicaBizau/cdnjs-importer | https://github.com/cdnjs/cdnjs-importer 

### Other Corrected URLs 
Was | Now 
--- | --- 
http://www.jgallardo.me/ | http://jgallardo.me/ 
https://cdnjs.cloudflare.com | https://cdnjs.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ Donation:
 ﻿[![tip for next commit](https://tip4commit.com/projects/919.svg)](https://tip4commit.com/github/cdnjs/cdnjs) [![Bountysource](https://www.bountysource.com/badge/team?team_id=11914&style=bounties_posted)](https://www.bountysource.com/teams/cdnjs/bounties?utm_source=cdnjs&utm_medium=shield&utm_campaign=bounties_posted)
 
 
-[cdnjs](https://github.com/cdnjs/cdnjs) is the repository mirroring all library assets on [cdnjs.cloudflare.com](https://cdnjs.cloudflare.com).
+[cdnjs](https://github.com/cdnjs/cdnjs) is the repository mirroring all library assets on [cdnjs.cloudflare.com](https://cdnjs.com/).
```
Comment (PeterDaveHello) [inline README.md]: I don't wanna have this change.

Comment (ReadmeCritic) [inline README.md]: Will update

**Comment by ReadmeCritic** [comment]
Ready

**Comment by PeterDaveHello** [comment]
Would you please squash/fixup these commits into only one commit as our document says? Thanks.

**Comment by ReadmeCritic** [comment]
Yes

**Comment by ReadmeCritic** [comment]
Oops..

**Pull request closed without merging**